Libraries/Storage/AsyncStorage.js

```diff
@@ -480,6 +480,23 @@ var AsyncStorage = {
       });
     });
   },
+
+  /**
+   * Call this set the storage location.
+   * @param storageLocation The storage location. Can be 'documents' or 'applicationSupport'.
+   *
+   * **Note**: changing this only sets the location for future actions and does not migrate
+   * the store from the previous location.
+   *
+   * @platform ios
+   */
+  setStorageLocation: function(storageLocation: string) {
```

Comment:
`setStorageLocationIOS` makes sense if there will never be equivalent functionality on android cc @satya164